.circleci/config.yml

```diff
@@ -1397,7 +1397,8 @@ jobs:
       - matrix_notify_failure_unless_pr
 
   t_ems_ext_edr:
-    <<: *base_node_small
+    # Runs OOM on the small instance
```

Comment:
You should take the env from that image too (i.e. update the `base_node_small_env` below).

But actually, I'm surprised this even works without further changes. I'd expect the plain ubuntu image to be missing some JS stuff.

Reply:
Fixed!